invenio_rdm_records/services/review/service.py

```diff
@@ -143,19 +145,53 @@ def delete(self, identity, id_, revision_id=None, uow=None):
         return True
 
     @unit_of_work()
-    def submit(self, identity, id_, data=None, revision_id=None, uow=None):
-        """Submit record for review."""
-        # Get record and check permission
-        draft = self.draft_cls.pid.resolve(id_, registered_only=False)
-        self.require_permission(identity, "update_draft", record=draft)
+    def submit(
+        self, identity, id_, data=None, require_review=False, revision_id=None, uow=None
+    ):
+        """Submit record for review or direct publish to the community."""
+        if not isinstance(require_review, bool):
+            raise ValidationError(
+                _("The require_review param must be true or false."),
```

Comment:
`require_review` is already in the `field_name`, and _param_ seems very code oriented.
```suggestion
                _("Must be a boolean, True or False"),
```